src/python/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+find_package(Python 3.9
```

Comment:
Could add your name up there if you'd like :D!

Reply:
Added header